cmake/packaging/debian.cmake

```diff
@@ -179,6 +179,16 @@ macro(ov_cpack_settings)
         _ov_add_plugin(gpu OFF)
         set(gpu_copyright "generic")
     endif()
+    
+    # openvino-nvidia-gpu
+    if(ENABLE_NVIDIA)
```

Comment:
I suppose we need to use `BUILD_nvidia_plugin` condition (see https://github.com/openvinotoolkit/openvino_contrib#how-to-build-openvino-with-extra-modules for reference)